.github/copilot-instructions.md

```diff
@@ -76,6 +78,7 @@ Before applying any changes, ensure you have a full successful build of the need
     - **Mono Runtime:** `./build.sh mono+libs`
     - **Libraries:** `./build.sh clr+libs -rc release`
     - **WASM/WASI Libraries:** `./build.sh mono+libs -os browser`
+    - **Host:** `./build.sh clr+libs+host -rc release -lc release`
```

Comment:
Do we always want `release`?

Reply:
I was going to ask the same thing, but then noticed the other instructions have `release` as well.

Reply:
Libraries does, but CLR is `clr+libs+host`, which is debug, right?